### PR TITLE
Optionally allow specified finite value as cutoff in featureset imputation

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

File: cesium/featureset.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def __repr__(self):
         s = xr.Dataset.__repr__(self)
         return s.replace('<xarray.', '<cesium.')
 
-    def impute(self, strategy='constant', value=None):
+    def impute(self, strategy='constant', value=None, max_mag=np.inf):
```
Comment (bnaul) [inline cesium/featureset.py]: Call this something generic (`max_value`?).

File: cesium/featureset.py [hunk 2]
```diff
@@ -47,12 +47,16 @@ def impute(self, strategy='constant', value=None):
             `None`, in which case a very large negative value is used (a
             good choice for e.g. random forests).
 
+        max_mag : float or `np.inf`, optional
```
Comment (bnaul) [inline cesium/featureset.py]: Just `float`

File: cesium/featureset.py [hunk 3]
```diff
@@ -47,12 +47,16 @@ def impute(self, strategy='constant', value=None):
...
+            Maximum absolute magnitude at which point value is imputed.
```
Comment (bnaul) [inline cesium/featureset.py]: `magnitude` -> `value`

**Comment by bnaul** [comment]
Made a little docstring tweak, 👍  from me. @stefanv wanna merge?

File: cesium/featureset.py [hunk 4]
```diff
@@ -27,7 +27,7 @@ def __repr__(self):
         s = xr.Dataset.__repr__(self)
         return s.replace('<xarray.', '<cesium.')
 
-    def impute(self, strategy='constant', value=None):
+    def impute(self, strategy='constant', value=None, max_value=1.e32):
```
Comment (stefanv) [inline cesium/featureset.py]: 1e32 is probably a slightly less confusing syntax here

File: cesium/featureset.py [hunk 5]
```diff
@@ -47,12 +47,16 @@ def impute(self, strategy='constant', value=None):
             `None`, in which case a very large negative value is used (a
             good choice for e.g. random forests).
 
+        max_value : float, optional
+            Maximum (absolute) value above which values are treated as infinite.
```
Comment (stefanv) [inline cesium/featureset.py]: What is the rationalization for this parameter?  (I guess it is related to the scikit-learn discussion we had the other day, but may be worth outlining here in some more detail.)

**Pull request closed without merging**